src/components/Featured/Featured.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import MovieCard from "../MovieCard/MovieCard";
+import './Featured.css';
+
+const Featured = (props) => {
+  const featuredCards = [...props.movies].sort((a, b) => b.average_rating - a.average_rating).map(movie => <MovieCard title={movie.title} averageRating={movie.average_rating} posterPath={movie.poster_path} selectMovie={props.selectMovie} id={movie.id} key={movie.id} /> )
```

Comment:
Good use of the spread operator!

Reply:
If this application was working with a huge amount of movies (like, thousands), I might encourage you to rethink your approach here.   I do not think this need refactored since you're NOT working thousands of movies.  But consider, since you only care about rendering 4 movies in the end, if your movies array did hold thousands of movies, you might want to sort it, find a way to isolate the 4 you care about, then map over to turn only those 4 into <MovieCard> components.